_docs/tasks/security/authn-policy.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Basic Istio authentication policy
+overview: This task shows you how to use Istio authentication policy to setup mutual TLS and simple end-user authentication.
+
+order: 10
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+Through this task, you will learn how to:
+
+* Using authentication policy to setup mutual TLS.
+
+* Using authentication policy to end-user authentication.
+
+
+## Before you begin
+
+* Understand Isio [authentication policy]({{home}}/docs/concepts/security/authn-policy.html) and related [mutual TLS authentication]({{home}}/docs/concepts/security/mutual-tls.html) concepts.
+
+* Know how to verify mTLS setup (recommend to walk through [testing Istio mutual TLS authentication]({{home}}/docs/tasks/security/mutual-tls.html))
+
+* Have a Kubernetes cluster with Istio installed (either with or without mTLS, but preferred without as the tasks will show the way to enable mTLS with policy)
```

Comment:
Can we give an instruction "without mTLS" instead of saying "with or without mTLS"?

Reply:
Done